modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1834,7 +1834,11 @@ AUI.add(
 							layoutValue && layoutValue.privateLayout
 						);
 
-						var groupId = themeDisplay.getScopeGroupIdOrLiveGroupId();
+						var groupId = themeDisplay.getScopeGroupId();
+
+						if (themeDisplay.isStagedPortlet() === 'false') {
```

Comment:
We can make this clause just `if(themeDisplay.isStagedPortlet())` if the return of the isStagedPortlet function is a boolean like so:
```
return <%= liveGroup.isStagedPortlet(themeDisplay.getPpid()) %>; 
```

Reply:
Hey, @bakayattila. It is a string because [here](https://github.com/liferay-forms/liferay-portal/pull/1534/files#diff-85bb8302271a11c1e5ab780bbea0be403a73665d742def70c0d817f8d43e6713R315) you're wrapping the result in single quotes. Please remove those so the type returned is a boolean.